### PR TITLE
Static site changes

### PR DESCRIPTION
**Pull request by jfaz1**
Some ideas to get the ball rolling on the homepage.

### CODE REVIEW

File: static/src/pages/index.astro [hunk 1]
```diff
@@ -13,27 +13,27 @@ export const prerender = true;
...
   <Hero>
-    <Fragment slot="subtitle">Modern Infrastructure</Fragment>
-    <Fragment slot="title">for Ambitious Product Teams</Fragment>
+    <Fragment slot="subtitle">The ultimate platform</Fragment>
+    <Fragment slot="title">for modern service development.</Fragment>
     <Fragment slot="content">
-      Finally, a platform that can bring any product idea to life in seconds.
-      Batteries Included.
+      Experience unparalleled flexibility and control. Build, deploy, and scale
+      your projects with ease using our open-source, all-in-one solution.
```
Comment (elliottneilclark) [inline static/src/pages/index.astro:28]: Should we say source available here?

**Review by elliottneilclark — APPROVED** [review]

**Comment by jfaz1** [comment]
whoops. lemme rebase.

**Review by elliottneilclark — APPROVED** [review]

**Pull request merged**